<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerMenu</class>
 <widget class="QWidget" name="PlayerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>34</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Memento</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menuMedia">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Media</string>
      </property>
      <widget class="QMenu" name="menuRecentFiles">
       <property name="title">
        <string>Open Recent</string>
       </property>
       <addaction name="separator"/>
       <addaction name="actionClearRecents"/>
      </widget>
      <addaction name="actionOpen"/>
      <addaction name="actionOpenUrl"/>
      <addaction name="menuRecentFiles"/>
     </widget>
     <widget class="QMenu" name="menuAudio">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Audio</string>
      </property>
      <addaction name="actionAudioNone"/>
     </widget>
     <widget class="QMenu" name="menuSubtitle">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Subtitle</string>
      </property>
      <widget class="QMenu" name="menuSubtitleTwo">
       <property name="title">
        <string>Second Track</string>
       </property>
       <addaction name="actionSubtitleTwoNone"/>
      </widget>
      <widget class="QMenu" name="menuSizePosition">
       <property name="title">
        <string>Size and Position</string>
       </property>
       <addaction name="actionIncreaseSize"/>
       <addaction name="actionDecreaseSize"/>
       <addaction name="actionMoveUp"/>
       <addaction name="actionMoveDown"/>
      </widget>
      <addaction name="actionSubVis"/>
      <addaction name="menuSizePosition"/>
      <addaction name="actionSubtitlePause"/>
      <addaction name="actionShowSubtitleList"/>
      <addaction name="separator"/>
      <addaction name="actionAddSubtitle"/>
      <addaction name="menuSubtitleTwo"/>
      <addaction name="separator"/>
      <addaction name="actionSubtitleNone"/>
     </widget>
     <widget class="QMenu" name="menuSettings">
      <property name="geometry">
       <rect>
        <x>1033</x>
        <y>270</y>
        <width>200</width>
        <height>184</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QMenu" name="menuAnkiProfile">
       <property name="geometry">
        <rect>
         <x>1228</x>
         <y>274</y>
         <width>140</width>
         <height>52</height>
        </rect>
       </property>
       <property name="title">
        <string>Anki Profile</string>
       </property>
      </widget>
      <addaction name="menuAnkiProfile"/>
      <addaction name="actionOptions"/>
      <addaction name="separator"/>
      <addaction name="actionAbout"/>
      <addaction name="actionOpenConfig"/>
      <addaction name="actionUpdate"/>
     </widget>
     <widget class="QMenu" name="menuTools">
      <property name="title">
       <string>Tools</string>
      </property>
      <addaction name="actionShowSearch"/>
      <addaction name="actionOCRMode"/>
     </widget>
     <addaction name="menuMedia"/>
     <addaction name="menuAudio"/>
     <addaction name="menuSubtitle"/>
     <addaction name="menuTools"/>
     <addaction name="menuSettings"/>
    </widget>
   </item>
  </layout>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAudioNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionVideoNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSubtitleNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionSubtitleTwoNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionAddSubtitle">
   <property name="text">
    <string>Add Subtitle File</string>
   </property>
  </action>
  <action name="actionSubtitlePause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Memento</string>
   </property>
  </action>
  <action name="actionIncreaseSize">
   <property name="text">
    <string>Increase Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionDecreaseSize">
   <property name="text">
    <string>Decrease Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="text">
    <string>Open URL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionOpenConfig">
   <property name="text">
    <string>Open Config Folder</string>
   </property>
  </action>
  <action name="actionSubVis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Subtitles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionShowSearch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionShowSubtitleList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Subtitle List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOCRMode">
   <property name="text">
    <string>Enter OCR Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionClearRecents">
   <property name="text">
    <string>Clear Recents</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
